pkgs/by-name/sh/shanggu-fonts/package.nix

```diff
@@ -51,8 +51,8 @@ stdenvNoCC.mkDerivation {
     + lib.strings.concatLines (
       lib.lists.forEach (builtins.attrNames source) (
         name: (''
-          install -Dm444 ${name}/*.ttc -t $'' + name + ''/share/fonts/truetype
-          ln -s $'' + name + ''/share/fonts/truetype/*.ttc $out/share/fonts/truetype
+          install -Dm444 ${name}/*.ttc -t ''$${name}/share/fonts/truetype
+          ln -s ''$${name}/share/fonts/truetype/*.ttc $out/share/fonts/truetype
           ''
```

Comment:
That still looks off 😅

Reply:
The language is at fault

Reply:
I improved readability here: https://github.com/NixOS/nixpkgs/pull/350414

Reply:
fixed.